packages/extension-chakra-storefront/src/hooks/use-einstein.js

```diff
@@ -13,7 +13,7 @@ import {
     useEncUserId,
     useCustomerType
 } from '@salesforce/commerce-sdk-react'
-import {useConfig} from '../hooks/use-config'
+import {useConfig} from '../hooks/use-extension-config'
```

Comment:
In this file and other files too, let's make sure the hook name is correct.

```suggestion
import {useExtensionConfig} from '../hooks/use-extension-config'
```

Reply:
thanks! they should be fixed